src/main/java/wayoftime/bloodmagic/common/living/LivingHelper.java

```diff
@@ -61,7 +65,11 @@ public static boolean isNeverValid(Player player) {
     }
 
     public static boolean isNeverValid(ItemStack plate) {
```

Comment:
not related to your changes but `isNeverValid` might need a better function name